src/components/WorkOrder/MobileWorkingWorkOrderView.tsx

```diff
@@ -222,53 +233,67 @@ const MobileWorkingWorkOrderView = ({ workOrderReference }: Props) => {
     )
 
     try {
-      const filesToUpload = workOrderFiles.concat(followOnFiles)
       setCloseFormValues({
         workOrderFiles: workOrderFiles,
         followOnFiles: followOnFiles,
       })
 
-      if (filesToUpload.length > 0) {
-        if (workOrderFiles.length > 0) {
-          const uploadResult = await uploadFiles(
-            workOrderFiles,
-            workOrderReference,
-            data.workOrderPhotoDescription,
-            'Closing work order',
-            setLoadingStatus
+      cwLogger.log(`Uploading | ${filesToUpload.length} files`)
+
+      const fileGroups: {
+        files: File[]
+        description: string
+        label: string
+      }[] = [
```

Comment:
This data type is confusing to read.
I'd just define it separately above & then reference here.